pkgs/top-level/all-packages.nix

```diff
@@ -14114,6 +14121,8 @@ with pkgs;
 
   fluidd = callPackage ../applications/misc/fluidd { };
 
+  flutter-engine = callPackage ../development/libraries/flutter-engine {};
+
   flutterPackages =
     recurseIntoAttrs (callPackage ../development/compilers/flutter { });
   flutter = flutterPackages.stable;
```

Comment:
stupid question:
what is the difference between flutter and flutter-engine?

Reply:
Flutter is for the `flutter` command which you use for managing and building applications. Flutter engine is the actual engine which has the DartVM and actual code for running a Flutter program.

Reply:
Basically https://github.com/flutter/flutter vs https://github.com/flutter/engine.

Reply:
makes sense ^^

so the engine description should be something like

>  Flutter runtime with Flutter core libraries, Dart runtime and Dart compiler

Reply:
Well, it doesn't really have its core libraries. I think the original description or the GitHub repo's description is better than that.

Reply:
I'll just use this line from the README:
> The [Flutter](https://flutter.dev/) Engine is a portable runtime for hosting Flutter applications.